client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.tsx

```diff
@@ -0,0 +1,35 @@
+import { Gridicon } from '@automattic/components';
+import { translate } from 'i18n-calypso';
+import React from 'react';
+import poweredByTitanLogo from 'calypso/assets/images/email-providers/titan/powered-by-titan.svg';
+import { getTitanProductName } from 'calypso/lib/titan';
+import { ProviderCard } from 'calypso/my-sites/email/email-providers-stacked-comparison';
+
+import './professional-email-card.scss';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noop = () => {};
+const logo = <Gridicon className="professional-email-card__logo" icon="my-sites" />;
+const badge = <img src={ poweredByTitanLogo } alt={ translate( 'Powered by Titan' ) } />;
+const getTitanFeatures = () => {
+	return [
+		translate( 'Inbox, calendars, and contacts' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Email, calendars, and contacts' )` *ES Score: 6*